### ISSUE TITLE
Is it possible to offer a block list to the generic phone?

### ISSUE THREAD

**Issue by raarts** [ianlin/react-native-callkit#12]
I want to block specific numbers (and send them to voicemail), both on the VoIP as well as on cell calls. Is that possible?

**Comment by ianlin** [comment]
Yeah it's possible but haven't implemented since I don't have this requirement yet. PR is welcome!

**Comment by raarts** [comment]
Do you have build instructions? Can you give me hints on how to teach myself about this as fast as possible? Thanks.
